app/v1/models/fullReturn/FullReturnModel.scala

```diff
@@ -61,12 +57,24 @@ case class FullReturnModel(agentDetails: AgentDetailsModel,
     _.zipWithIndex.map(x => FullReturnModel.deemedParentCrnPath(x._2) -> x._1)
   }
 
+  val numberOfUkCompanies: Int = ukCompanies.length
+  val aggregateNetTaxInterestIncome: BigDecimal = totalTaxInterestIncome - totalTaxInterestExpense
```

Comment:
I think this is overcomplicated.

Can't you just have `aggregateNetTaxInterest` which is `totalTaxInterestIncome - totalTaxInterestExpense`
If this is negative then output in writes as `aggregateNetTaxInterestExpense` else `aggregateNetTaxInterestIncome`?